cmd/operator/deploy/crds/monitoring.googleapis.com_operatorconfigs.yaml

```diff
@@ -103,6 +103,34 @@ spec:
               object represents. Servers may infer this from the endpoint the client
               submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
             type: string
+          managedAlertmanager:
```

Comment:
A bit odd that this has a single key `config`. Any reason to not just move the properties of `config` up a level and remove the `config` key?

Reply:
I like that this leaves room to add more settings for the alertmanager. `config` here is referring to the actual config file, but I could see other settings under `AlertmanagerSpec` as well (replicas, auth..)